config/routes.rb

```diff
@@ -304,6 +304,10 @@
       get '/capture_doc/return_to_sp' => 'capture_doc#return_to_sp'
       get '/capture_doc/:step' => 'capture_doc#show', as: :capture_doc_step
       put '/capture_doc/:step' => 'capture_doc#update'
+
+      # deprecated routes
+      get '/confirmations' => 'personal_key#show'
+      post '/confirmations' => 'personal_key#update'
```

Comment:
```suggestion
      post '/confirmations' => 'personal_key#update'
      get '/download_personal_key' => 'confirmations#download'
```
Do we need this one?

Reply:
`/download_personal_key` [still exists](https://github.com/18F/identity-idp/pull/5785/files#diff-959bc9abc46a55332bb64d5155a79323afa75a50ec1a2137ddd22d926f62c6c5R268) but has been updated to the new controller name

Reply:
ah right, thanks!